doc/content/gateways/gateway-claiming/claim-gateways.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Claim a Gateway"
+description: ""
+weight: 2
+distributions: ["Enterprise", "Cloud"]
+--- 
+
+You can update the example below with your tenant ID and cluster ID by filling them here.
+
+{{< tenant-cluster-selector >}}
+
+## Prerequisites
+
+1. An account in The Things Industries Cloud.
+2. EUI of the gateway that you wish to claim.
+3. The Claim Authentication Code for the gateway (shared out of band by the current gateway owner).
+4. The Current CUPS Key used by the gateway (for {{% lbs %}} only).
```

Comment:
```suggestion
1. A user account in The Things Industries Cloud
2. EUI of the gateway that you wish to claim
3. The Claim Authentication Code for the gateway (shared out of band by the current gateway owner)
4. The current CUPS Key used by the gateway (for {{% lbs %}} only)
```

Reply:
Why The Things Industries Cloud? In other file you're using The Things Stack Cloud, but I'm confused with that too since labels tell The Things Stack Cloud and Enterprise. Please be consistent with these (or tell me if I'm misunderstanding something)

Reply:
Yeah you're right. This is corrected.